benchmarks/ampi/alltoall/Makefile

```diff
@@ -23,11 +24,11 @@ alltoall.mpi: alltoall.c
 
```

Comment:
Should we run these tests? (And also alltoall_VPtest).

Reply:
As long as they don't take an excessive amount of time to complete, I think that would be good.

Reply:
`alltoall` and `allgather` test are crashing on my machine, so I haven't added them yet. I've opened this issue for that (https://github.com/UIUC-PPL/charm/issues/2974). 

I added `alltoall_VPtest` to both test and testp in the latest commit.